cmd/openshift-install/create.go

```diff
@@ -655,3 +717,177 @@ func checkIfAgentCommand(assetStore asset.Store) {
 		logrus.Warning("An agent configuration was detected but this command is not the agent wait-for command")
 	}
 }
+
+func getClusterOperatorNames(ctx context.Context, cc *configclient.Clientset) ([]string, error) {
+	listCtx, cancel := context.WithTimeout(ctx, 1*time.Minute)
```

Comment:
I don't understand this 1m timeout.  It seems unlikely that the list takes longer than that, but if it does, and the wrapping `ctx` has more time available, wouldn't we want to wait longer?